provider/zencoder/zencoder.go

```diff
@@ -20,13 +20,14 @@ import (
 	"net/url"
 	"path"
 	"strconv"
+	"strings"
 	"time"
 
 	"github.com/NYTimes/video-transcoding-api/config"
 	"github.com/NYTimes/video-transcoding-api/db"
 	"github.com/NYTimes/video-transcoding-api/db/redis"
 	"github.com/NYTimes/video-transcoding-api/provider"
-	"github.com/brandscreen/zencoder"
+	"github.com/flavioribeiro/zencoder"
```

Comment:
if @brandscreen members accept https://github.com/brandscreen/zencoder/pull/4 we can move back to `brandscreen/zencoder` lib